config_utilities_ros/setup.cfg

```diff
@@ -0,0 +1,4 @@
+[develop]
```

Comment:
This shouldn't be needed unless you're trying to use the `setup.py` ament python package version (which you shouldn't need given you're using `ament_cmake_python`)

Reply:
To quickly clarify: the develop tag or the setup.cfg file? The setting up things for python and C++ was a bit weird, so not unlikely I duplicated/have some artifact parts.